### DISCUSSION TITLE
Unsure on Python code for Fluent setup

### DISCUSSION THREAD

**Discussion by ToMdAvI** [ansys/pyfluent#2535] [Q&A]
### 🔍 Before submitting the issue

- [X] I have searched among the existing issues
- [X] I am using a Python virtual environment

### 🐞 Description of the bug

1 - I am struggling to define my inlet velocity, I would like to change to components and set my x-velocity.
2 - How do I define my reference value?
3 - How do I add report definitions for drag and lift?
![image](https://github.com/ansys/pyfluent/assets/153080647/b0948e69-002b-4d3a-bbbb-a165898afbd2)
![image-1](https://github.com/ansys/pyfluent/assets/153080647/585cc731-90c8-487c-9f44-e677a6a439e5)
![image-2](https://github.com/ansys/pyfluent/assets/153080647/85ce4bf5-61eb-4883-8d7d-cd1666ed8cc0)

### 📝 Steps to reproduce

...

### 💻 Which operating system are you using?

Windows

### 🐍 Which Python version are you using?

3.11

### 📦 Installed packages

```shell
python -m pip freeze
```

**Comment by raph-luc** [comment]
Please peruse the documentation: https://fluent.docs.pyansys.com/version/stable/index.html

In particular, work through these examples: 
https://fluent.docs.pyansys.com/version/stable/examples/00-fluent/exhaust_system_settings_api.html (e.g., see velocity inlet)
https://fluent.docs.pyansys.com/version/stable/examples/00-fluent/mixing_elbow_settings_api.html (e.g., see report definition)
